### PR TITLE
delete the invalid case in  verifyAndGetSecretNameAndNamespaceTemplate

### PR DESCRIPTION
**Pull request by jianglingxia**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

> /kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes # the invalid case in function verifyAndGetSecretNameAndNamespaceTemplate

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -145,31 +145,31 @@ var (
 	}
 
 	provisionerSecretParams = secretParamsMap{
-		name:                         "Provisioner",
+		name: "Provisioner",
```
Comment (chrishenzie) [inline pkg/controller/controller.go:148]: Did you mean to change this indentation? I'm pretty sure `gofmt` will add this back when its ran

**Review by chrishenzie — CHANGES_REQUESTED** [review]
/ok-to-test

File: pkg/controller/controller.go (hunk 1 above)
Comment (jianglingxia) [reply pkg/controller/controller.go:148]: I run gofmt in normal format, but the file is indented when I save it

**Comment by jianglingxia** [comment]
@chrishenzie can you help to review this commit? thanks

**Comment by chrishenzie** [comment]
Hi, thanks for the ping. Quick question, I have less context here, can you provide context why this case is not possible? Also, can you squash your indentation commit into the previous one?

**Pull request closed without merging**